### DISCUSSION TITLE
postal initialize

### DISCUSSION THREAD

**Discussion by mrik007** [postalserver/postal#2044] [Help with using Postal]
Im getting an error while putting the code "postal initialize" 

Please help
![postal initialize error](https://user-images.githubusercontent.com/108406140/176430842-0dec8a3c-42c2-40cc-a400-ecaa7826d968.png)

**Comment by willpower232** [comment]
the first line of the trace, which isn't in your screenshot, would say exactly what the problem was however we can see that "db:create" is mentioned which means that is probably where the problem lies. It could be incorrect details in the config file or that the database isn't up and working yet.

**Reply by mrik007** [reply]
Hi, im getting an error while using MariaDB and then using postal initialize, postal make-user. 

Please check the attachment.
![docker](https://user-images.githubusercontent.com/108406140/176435724-97adc695-f8e9-4131-8d49-0813eba3d998.png)
![postal initialize error1](https://user-images.githubusercontent.com/108406140/176435734-32a26e28-90c1-4379-af19-f2c6e3cc2cf1.png)
![postal make user](https://user-images.githubusercontent.com/108406140/176435738-250a9a78-99f7-40a0-81f5-2688278d6564.png)
![rabbitmq](https://user-images.githubusercontent.com/108406140/176435743-daedcccb-a7b0-4716-a118-1ce4a2d60159.png)

**Reply by willpower232** [reply]
> can't connect to database server on 127.0.0.1

can you see it running? can you check it has the port with `netstat -lnp`

**Reply by mrik007** [reply]
Yes I can so how to fix this issue??? pls guide

**Reply by mrik007** [reply]
Im using codes from here - https://inguide.in/build-postal-smtp-server-send-bulk-emails-ubuntu-18-04/

**Reply by mrik007** [reply]
Hi, 
i finally completed the putty setup but I'm getting this error 
![errr](https://user-images.githubusercontent.com/108406140/176456468-8c27df6a-823e-4298-8a50-3356fd29eeb6.png)

**Reply by willpower232** [reply]
You will need to `postal app bash` and then have a look in /opt/postal/logs or something like that to see what the problem is.
